elk/files.py

```diff
@@ -38,3 +38,8 @@ def memorably_named_dir(parent: Path):
     out_dir = parent / sub_dir
     out_dir.mkdir(parents=True, exist_ok=True)
     return out_dir
+
+
+def transfer_eval_directory(source: str) -> Path:
+    """Return the directory where transfer evals are stored."""
+    return elk_reporter_dir() / source / "transfer_eval"
```

Comment:
because tests need to know where it is
(and also we previously mispelled it as `transfer_evals` (with an s) rather than `transfer_eval`, haha